Source/NSMenuItemCell.m

```diff
@@ -290,6 +290,8 @@ - (NSString*) _keyEquivalentString
                           (shift != NO) ? shiftKeyString : @"",
                           (m & NSCommandKeyMask) ? commandKeyString : @"",
                           key];
+
+	  key = [[GSTheme theme] keyForKeyEquivalent: key];
```

Comment:
Again I am wondering whether we should move more of the original code into the default theme. As the code is now, the theme must be fully aware on how we build up the key here, parse that and rewrite it.
The only thing that speaks against that move is that this code here also changes the key equivalent font. So it is not side effect free.

Reply:
I think you're right, as much as possible of the "default" drawing behaviour should be moved into the default GSTheme class, both for architectural reasons, and because it affords theme authors far 
more power and flexibility in customizing the visual presentation.

So perhaps the implementation of `_keyEquivalentFont` and `_setKeyEquivalentFont` should defer to the theme, as well as replacing my implementation of `GSTheme-[keyForKeyEquivalent]` with the existing body of `NSMenuItemCell-[_keyEquivalentString]`, and have that method itself defer to the matching theme method?

Reply:
I fully agree with your comment, but your code does not reflect this.

Reply:
I was waiting for feedback on the idea before implementing it -- but I'm glad we're on the same page there. I will get that done, probably in my next PR, which I plan to be specifically about moving more default behaviour into the base GSTheme class, as opposed to the minimal set of hooks I needed for my own personal passion project.